templates/controllers/grid/grid.tpl

```diff
@@ -83,7 +81,7 @@
 				so that we can restore it if the user deletes all rows.
 			**}
 			<tr>
-				<td colspan="{$grid->getColumnsCount('indent')|@count}">{translate key=$grid->getEmptyRowText()}</td>
```

Comment:
This is just for the body. The grid.tpl file is used by both, Grid and CategoryGrid controllers.